src/metabase/core.clj

```diff
@@ -163,6 +164,8 @@
   (settings/migrate-encrypted-settings!)
   ;; start scheduler at end of init!
   (task/start-scheduler!)
+  ;; In case we could not do this earlier, because the scheduler was not up yet:
```

Comment:
nit - usage context might be helpful here for the future
```suggestion
  ;; In case we could not do this earlier (i.e. for DBs added via config file), because the scheduler was not up yet:
```

Reply:
https://github.com/metabase/metabase/pull/48882/commits/e09b6ff5088a52a98cdbf388b1d1241c33fdb323